mentat/config.py

```diff
@@ -83,6 +83,13 @@ class Config:
         },
         converter=converters.optional(converters.to_bool),
     )
+    auto_tokens: int = attr.field(
+        default=4000,
```

Comment:
I think the default should be a little higher. Like 8000. The one benchmark I wrote fails when autocontext is this low but succeeds at 8000. Sorry I wish we had more benchmarks already so we could be more scientific.

Reply:
ya I don't know either. I'll set it to 8k and lets keep an eye on it.